### ISSUE TITLE
Launch error as of 1.3.8

### ISSUE THREAD

**Issue by Jiralite** [MCMi460/NSO-RPC#91]
I'm using a MacBook Pro (M1) running macOS Ventura 13.4.1. As of 1.3.8, this application does not start and simply states the following:

<img width="372" alt="Launch error" src="https://github.com/MCMi460/NSO-RPC/assets/33201955/8223f5cd-611e-49d9-ad4e-18ee43119afa">

Downgrading to version 1.3.7 or version 1.3.6 worked fine. I restarted my laptop as a sanity check too. Upgrading again to version 1.3.8 yields the error once more. I have tried installing both mac-installer.dmg and mac-portable.zip to no avail. Running `/Applications/NSO-RPC.app/Contents/MacOS/NSO-RPC` yields the following:

```xl
jiralite@Jiralites-MacBook-Pro ~ % /Applications/NSO-RPC.app/Contents/MacOS/NSO-RPC
Traceback (most recent call last):
  File "/Applications/NSO-RPC.app/Contents/Resources/__boot__.py", line 170, in <module>
    _run()
  File "/Applications/NSO-RPC.app/Contents/Resources/__boot__.py", line 84, in _run
    exec(compile(source, path, "exec"), globals(), globals())
  File "/Applications/NSO-RPC.app/Contents/Resources/app.py", line 3, in <module>
    from qtpy.QtCore import *
  File "qtpy/__init__.pyc", line 259, in <module>
qtpy.QtBindingsNotFoundError: No Qt bindings could be found
2023-07-17 01:32:33.691 NSO-RPC[2841:29073] Launch error
2023-07-17 01:32:33.691 NSO-RPC[2841:29073] Launch error
See the py2app website for debugging launch issues
```

**Comment by MCMi460** [comment]
Same thing here! Going to have to check through the recent code, as the Qt-related stuff isn't really my expertise, but have you tried the universal one? It works much better for M1s.

**Comment by Jiralite** [comment]
> have you tried the universal one? It works much better for M1s.

Oh really? I didn't know that. I'll definitely start using it then if it's better for my device! I tried it out now and it works just fine. Thank you for that recommendation!

**Comment by MCMi460** [comment]
This should be fixed + clarified in release [v1.3.9](https://github.com/MCMi460/NSO-RPC/releases/tag/v1.3.9).
